.github/fabricbot.json

```diff
@@ -2196,7 +2196,599 @@
           }
         ]
       }
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "IssuesOnlyResponder",
+      "version": "1.0",
+      "config": {
+        "conditions": {
+          "operator": "and",
+          "operands": [
+            {
+              "name": "isAction",
+              "parameters": {
+                "user": "roubachof",
+                "action": "opened"
+              }
+            },
+            {
+              "operator": "or",
+              "operands": [
+                {
+                  "name": "isActivitySender",
+                  "parameters": {
+                    "user": "roubachof"
+                  }
+                },
+                {
+                  "name": "isActivitySender",
+                  "parameters": {
+                    "user": "davidbritch"
+                  }
+                }
+              ]
+            }
+          ]
+        },
+        "eventType": "issue",
+        "eventNames": [
+          "issues",
+          "project_card"
+        ],
+        "taskName": "Add 'i/great-reporter' when issue is opened by an author we know opens high quality issues",
+        "actions": [
+          {
+            "name": "addLabel",
+            "parameters": {
+              "label": "i/great-reporter"
+            }
+          }
+        ]
+      }
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "IssuesOnlyResponder",
+      "version": "1.0",
+      "config": {
+        "conditions": {
+          "operator": "and",
+          "operands": [
+            {
+              "name": "labelAdded",
+              "parameters": {
+                "label": "s/try-latest-version"
```

Comment:
Added a "please try latest version" label so we can easily instruct people to please try again on the latest version. Please review the description just below if it's clear enough or it needs work.

Reply:
I love this. We need to be careful to apply it only to issues that are fixed in _public_ releases, as opposed to "we merged the fix last night so good luck building the MAUI repo and using it in your app."